.pep8speaks.yml

```diff
@@ -0,0 +1,6 @@
+scanner:
+    diff_only: True
+    linter: flake8
+
+black:
```

Comment:
this is not a valid config, as the pep8 errors haven't been cleared by pep8speaks. I think this needs to be `flake8` instead of `black`

Reply:
Ouch... I forgot to change that, thanks for catching it. I had used black as default but then decided to use flake8 because of some E501 I need to be caught.